src/pages/components/Sidebar.tsx

```diff
@@ -40,8 +40,12 @@ const Sidebar = ({ items, collapsed }: SidebarProps) => {
         navigate(data.key);
     }
 
+    const menuStyle = { 
+        backgroundColor: '#001529',
```

Comment:
for this value can we move the color into the ColorsConstants? if doesn't exists you can create a new one just to have one file with all the colors in the code side (Just hardcoded code for the tailwing.css file nope)

Reply:
@imadiaz 
I have created a file to have the colors as constants so that only that file is modified instead of the two where ant is used
![constantColor](https://github.com/cDentalCareGroup/web-cdentalcaregroup/assets/83984948/d37dcdd7-2e58-4b20-871c-c9350dbacb8c)
![constantcolor2](https://github.com/cDentalCareGroup/web-cdentalcaregroup/assets/83984948/c627fb7f-0891-4ef8-bdee-58a68f11e14e)
![constantColorFile](https://github.com/cDentalCareGroup/web-cdentalcaregroup/assets/83984948/8e9f7f39-e43e-4cf8-8785-34935d550fb4)

Reply:
why constant SECONDARY_COLOR is the colorPrimary? I mean should be the same one right?